### PR TITLE
Get record stream header info from command line parameter

### PR DESCRIPTION
**Pull request by JeffreyDallas**
Resolve regression issue #3085

https://github.com/swirlds/swirlds-platform-regression/issues/3085

Test result

https://swirldslabs.slack.com/archives/C03G7CBJJ06/p1673991616079089

### CODE REVIEW

File: hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java [hunk 1]
```diff
@@ -99,6 +102,8 @@ public class FileSignTool {
     private static final String ALIAS_PROPERTY = "alias";
     private static final String PASSWORD_PROPERTY = "password";
     private static final String DIR_PROPERTY = "dir";
+    private static final String APP_VERSION = "appVersion";
+
     private static final Logger LOGGER = LogManager.getLogger(FileSignTool.class);
```
Comment (jeromy-cannon) [inline hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java:107]: would you like to migrate this to slf4j?

File: hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java [hunk 2]
```diff
@@ -307,7 +312,33 @@ private static void createSignatureFileForRecordFile(
...
+            if (versions.length >= 3) {
+                try {
+                    fileHeader =
+                            new int[] {
+                                DEFAULT_RECORD_STREAM_VERSION,
+                                Integer.parseInt(versions[0]),
+                                Integer.parseInt(versions[1]),
+                                Integer.parseInt(versions[2]),
+                            };
+                } catch (NumberFormatException e) {
```
Comment (qnswirlds) [inline hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java:331]: nit: add `final`

**Review by qnswirlds — APPROVED** [review]

File: hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java (hunk 1 above)
Comment (JeffreyDallas) [reply hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java:107]: Probably need a new PR for replacing all log4j2 with slf4j on hedera repo

File: hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java (hunk 2 above)
Comment (JeffreyDallas) [reply hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java]: updated

**Review by qnswirlds — APPROVED** [review]

File: hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java (hunk 1 above)
Comment (jeromy-cannon) [reply hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java:107]: I was thinking that the we were going to just add it as we changed other stuff.  Nathan assigned it to all three of us.

**Review by jeromy-cannon — APPROVED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]

**Pull request merged**